lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -104,6 +104,16 @@
             - Wait until vCenter detects an IP address for the VM
             - This requires vmware-tools (vmtoolsd) to properly work after creation
         default: False
+   snapshot_src:
+        description:
+            - Name of an existing snapshot to use to create a clone of a VM.
+        default: None
+        version_added: "2.4"
+   snapshot_linked:
```

Comment:
I have no qualms with that and it does sound a bit better.  I will submit the changes in a few minutes.